src/main.js

```diff
@@ -525,6 +527,51 @@ class MiniGraphCard extends LitElement {
       </svg>`;
   }
 
+  renderGridLines() {
+    const {
+      height, hours_to_show, show,
+    } = this.config;
+    const grid_lines_type = show.grid_lines_type ? show.grid_lines_type : false;
+    const grid_lines_ratio = (show.grid_lines_ratio && Number.isInteger(show.grid_lines_ratio))
+      ? show.grid_lines_ratio
+      : 2;
+
+    const containerWidth = 500;
+    let numLines;
+
+    let rounded_hours_to_show = Math.round(hours_to_show);
+    if (rounded_hours_to_show < 1) rounded_hours_to_show = 1;
+
+    switch (grid_lines_type) {
+      case '5minute':
+        numLines = rounded_hours_to_show * 12;
+        break;
+      case 'hour':
+      default:
+        numLines = rounded_hours_to_show;
+        break;
+      case 'day':
+        numLines = Math.round(rounded_hours_to_show / 24);
+        break;
+      case 'week':
+        numLines = Math.round(rounded_hours_to_show / 168);
+        break;
+    }
+    const xRatio = containerWidth / numLines;
+    const lines = [];
+
+    for (let i = 0; i < numLines; i += 1) {
+      const x = xRatio * (i + 0.5);
+      if (i % grid_lines_ratio > 0) {
+        lines.push(svg`<line x1=${x} y1="0" x2=${x} y2=${height} stroke="var(--mini-graph-card-thin-grid-color, var(--divider-color))" stroke-width="0.5"/>`);
+      } else {
+        lines.push(svg`<line x1=${x} y1="0" x2=${x} y2=${height} stroke="var(--mini-graph-card-thick-grid-color, rgb(from var(--divider-color) R G B /0.5))" stroke-width="0.5"/>`);
```

Comment:
Please replace CSS vars' names:
--mini-graph-card-thick-grid-color ->
--mcg-grid-line-thick-color

same for "thin" (just realized we already have `--mcg-title-letter-spacing` & `--mcg-title-font-size` supported)

Reply:
Also, please add at the end of "Theme variables" chapter of Docs these lines
| mcg-grid-line-thick-color | rgb(from var(--divider-color) R G B /0.5) | Grid "thick" line color.
| mcg-grid-line-thin-color | var(--divider-color) | Grid "thin" line color.

Reply:
Done.
I'm really not familiar with available css global variables.
Do you know a place where they are centralized ?

Reply:
There are plenty of css vars:
- added by 3rd party components used in HA (like Material) - example `--mdc-text-field-label-ink-color`;
- own HA frontend vars - example `--primary-text-color`;
- added by custom cards like mini-graph-card - example `--mcg-title-letter-spacing`;
- and users may add personal vars.

There is not place where all vars are centralized.
If you are looking for a tutorial related to CSS modding of HA frontend - check this [main card-mod thread](https://community.home-assistant.io/t/card-mod-add-css-styles-to-any-lovelace-card/120744) -> 1st post -> link at the bottom titled "fantastic" -> you will get a set of tutorials & examples.

Reply:
I'll take a look thanks